modules/custom/d_geysir/src/Utility/ModalDetector.php

```diff
@@ -46,12 +46,21 @@ public function __construct(RequestStack $request_stack, ControllerResolverInter
   public function isGeysirModalRequest(): bool {
     try {
       $controller = $this->resolver->getController($this->request);
+      $is_modal = (
```

Comment:
Hmmm to bym jednak wyciągał gdzieś do osobnej metody. 
Cała ta klasa jest tylko dla Ciebie, więc możęsz mieć conajmniej takie metody do sprawdzania:
`isModal` `isAajax` i `isNotOpElement`
Czy coś na tą modłę :-)
Możesz mieć do tego magiczną metodę typu `isGeysirModal` która zwraca ten warunek, w kazdym razie im bardziej granularnie tym łatwiej będzie to rozwijać lub zmieniac w zaleznosci od tego co tam sie w Drupalu kiedyś zamieni
Pomijam już fakt dwukrotnego pobierania `wrapper_format` ;-)